src/ui/textbox/textbox.ios.ts

```diff
@@ -135,7 +136,7 @@ export default class TextBoxIOS<TEvent extends string = TextBoxEvents, TNative =
           location: method.range,
           insertedText: method.replacementString
         });
-        return true;
+        return this.maxLength;
```

Comment:
We can remove this return statement, because, we are not getting the value from method return. We use property to check maxLength value in the framework @boztopuz